testutil/keeper/unit_test_helpers.go

```diff
@@ -195,6 +195,18 @@ func GenPubKey() (crypto.PubKey, error) {
 	return cryptocodec.ToTmPubKeyInterface(privKey.PrivKey.PubKey())
 }
 
+// Generates a public key for unit tests (abiding by tricky interface implementations from tm/sdk) or panics
+func MustGenPubKey() crypto.PubKey {
```

Comment:
Can be replaced by other existing key utils

Reply:
https://github.com/cosmos/interchain-security/commit/421bee2ff69b17d6b39a883ddc49dfc4fab5bdae